src/browser.ts

```diff
@@ -143,13 +157,21 @@ export class Browser {
   wsEndpoint() {
     return this.#celestial.ws.url;
   }
+
+  /**
+   * The browser's websocket ready state
+   */
+  wsReadyState() {
+    return this.#celestial.ws.readyState;
+  }
 }
 
 export interface LaunchOptions {
   headless?: boolean;
   path?: string;
   product?: "chrome" | "firefox";
   args?: string[];
+  browserWSEndpoint?: string;
```

Comment:
Could we maybe just change this to `endpoint`? This feels like a comedically long name. Maybe worth some bikeshedding considering this will be public API.

Reply:
Yeah initially I used this name because it was the one used by [puppeteer](https://pptr.dev/api/puppeteer.connectoptions/#properties), but I agree it feels super awkward 😅 

I switched it to `wsEndpoint` since astral already had `Browser.wsEndpoint` so this way it stays consistent in naming I think